sunkit_image/flct/flct.py

```diff
@@ -0,0 +1,499 @@
+    order : {"row" | "column"}
+        The order in which the array elements are stored that is whether they are stored as row
+        major or column major.
+
+    Returns
+    -------
+    A dat file of the ``filename`` is created in the current directory without returning anything.
+    """
+
+    # Checking whether the C extension is correctly built.
+    if _pyflct is None:
+        raise ImportError("C extension for flct is missing, please rebuild.")
+
+    if order.lower() not in ["row", "column"]:
+        raise ValueError("The order of the arrays is not correctly specified. It can only be 'row' or 'column'")
+
+    if order is "row":
+        transp = 0
+    else:
+        transp = 1
+
+    ier = _pyflct.write_three_images(filename, array1, array2, array3, transp)
+
+    if ier is not 1:
+        raise ValueError("The file was not read correctly. Please check the file")
+
+
+def column_row_of_two(array1, array2):
+    """
+    A function which takes two arrays and swaps the order in which they were stored i.e. changing
+    from column major to row major and ``not`` the vice-versa. This may change the values stored in
+    the array as the arrays are first converted to a binary format and then the order change takes
+    place.
+
+    Parameters
+    ----------
+    array1 : `numpy.ndarray`
+        The first array whose order is to be changed.
+    array2 : `numpy.ndarray`
+        The second array whose order is to be changed.
+
+    Returns
+    -------
+    `tuple`
+        It returns the two input arrays after changing their order from column major to
+        row major.
+    """
+
+    # Checking whether the C extension is correctly built.
+    if _pyflct is None:
+        raise ImportError("C extension for flct is missing, please rebuild.")
+
+    one, two = _pyflct.swap_order_two(array1, array2)
+
+    return (one, two)
+
+
+def column_row_of_three(array1, array2, array3):
+    """
+    A function which takes two arrays and swaps the order in which they were stored i.e. changing
+    from column major to row major and ``not`` the vice-versa. This may change the values stored in
+    the array as the arrays are first converted to a binary format and then the order change takes
+    place.
+
+    Parameters
+    ----------
+    array1 : `numpy.ndarray`
+        The first array whose order is to be changed.
+    array2 : `numpy.ndarray`
+        The second array whose order is to be changed.
+    array3 : `numpy.ndarray`
+        The third array whose order is to be changed.
+
+    Returns
+    -------
+    `tuple`
+        It returns the two input arrays after changing their order from column major to
+        row major.
+    """
+
+    # Checking whether the C extension is correctly built.
+    if _pyflct is None:
+        raise ImportError("C extension for flct is missing, please rebuild.")
+
+    one, two, three = _pyflct.swap_order_three(array1, array2, array3)
+
+    return (one, two, three)
+
+
```

Comment:
A general comment, can we not combine both the 2 and 3 array versions into one function instead of two?

Reply:
Yaa can do that but then I guess the inputs have to be a tuple